conf/suse/master

```diff
@@ -961,6 +963,28 @@ syndic_user: salt
 #
 #pillar_cache_backend: disk
 
+# A master can cache external pillars locally to bypass the expense of having to render them
+# for each minion on every request. This feature should only be enabled in cases
+# where external pillar rendering time is known to be unsatisfactory and any attendant security
+# concerns about storing external pillars in a master cache have been addressed.
+#
+# When enabling this feature, be certain to read through the additional ``ext_pillar_cache_*``
+# configuration options to fully understand the tunable parameters and their implications.
+#
+# Note: setting ``ext_pillar_cache: True`` has no effect on targeting Minions with Pillars.
+# See https://docs.saltstack.com/en/latest/topics/targeting/pillar.html.
+# And it also has no effect on pillar data rendered from pillar_roots.
+# See the additional ``pillar_cache_*`` configuration options.
+#ext_pillar_cache: False
+
+# If and only if a master has set ``ext_pillar_cache: True``, others same with 
+# pillar_cache_ttl.
+#ext_pillar_cache_ttl: 3600
+
+# If and only if a master has set ``ext_pillar_cache: True``, others same with 
+# pillar_cache_backend.
+#ext_pillar_cache_backend: disk
```

Comment:
This optioning is kind of scattered and also can be optimised. E.g. this would enable it and put TTL to 3600:

```yaml
pillar_cache:
   backend: disk
   ttl: 3600
```

Commenting all that out would be equal to disabling it.